### ISSUE TITLE
Privacy and Parallel Installation Issues.

### ISSUE THREAD

**Issue by farzbood** [letta-ai/letta#2238]
Well, I've no restrictions regarding **data-backend** to use, but **tight coupling** caused by attaching all `letta installations` **setup** to a **centralized location in User Profile folder of OS** (%USERPROFILE%/.letta/) is **restricting/conflicting** for scenarios like **A/B testing** that needs to have more than one` letta installation` (same and/or different versions) with **isolated setups/configurations** in parallel. (as I mentioned, I've more than one installations in isolated envs.)
How can we achieve that?
(Edit):
After upgrading to `0.6.3` , accessing `localhost:8283` instead of `ADE interface`, it loads a **modal dialog** redirecting me to a **Login Page** to the `letta cloud`, with no option to **opt-out**, why should I **login to letta cloud** for using a `local server` instance?!
![image](https://github.com/user-attachments/assets/54b95317-43ef-42e0-b359-3c22e50f1844)
![image](https://github.com/user-attachments/assets/2e754951-3c8f-452a-a031-418dc5339cce)

_Originally posted by @farzbood in https://github.com/letta-ai/letta/issues/2211#issuecomment-2534878175_

**Comment by cpacker** [comment]
> After upgrading to 0.6.3 , accessing localhost:8283 instead of ADE interface, it loads a modal dialog redirecting me to a Login Page to the letta cloud, with no option to opt-out, why should I login to letta cloud for using a local server instance?!

Copying over some answers from our FAQ here: https://github.com/letta-ai/letta?tab=readme-ov-file#-frequently-asked-questions-faq.

tl;dr We are not actively developing the old version of the ADE (that runs on localhost), however it is still freely available if you downgrade your Letta version. The new ADE is a web application intended to connect to both Letta servers in the cloud and locally, and currently requires a login to use while in open beta.
---

> _"How do I use the ADE locally?"_

To connect the ADE to your local Letta server, simply run your Letta server (make sure you can access `localhost:8283`) and go to [https://app.letta.com](https://app.letta.com). If you would like to use the old version of the ADE (that runs on `localhost`), downgrade to Letta version `<=0.5.0`.

> _"If I connect the ADE to my local server, does my agent data get uploaded to letta.com?"_

No, the data in your Letta server database stays on your machine. The Letta ADE web application simply connects to your local Letta server (via the REST API) and provides a graphical interface on top of it to visualize your local Letta data in your browser's local state.

> _"Do I have to use your ADE? Can I build my own?"_

The ADE is built on top of the (fully open source) Letta server and Letta Agents API. You can build your own application like the ADE on top of the REST API (view the documention [here](https://docs.letta.com/api-reference)).

**Comment by polytect** [comment]
Why the local UI was closed down with forced cloud UI? I mean seriously.

**Comment by cpacker** [comment]
> Why the local UI was closed down with forced cloud UI? I mean seriously.

https://docs.letta.com/guides/desktop/install
